Source/Models/FORMField.m

```diff
@@ -35,6 +35,12 @@ - (instancetype)initWithDictionary:(NSDictionary *)dictionary
     _subtitle = [dictionary andy_valueForKey:@"subtitle"];
     _typeString  = [dictionary andy_valueForKey:@"type"];
     _type = [self typeFromTypeString:self.typeString];
+    _inputType = [dictionary andy_valueForKey:@"input_type"];
+
+    if (_inputType) {
+        _typeString = _inputType;
```

Comment:
This part is odd, can you make it in a way that you don't overwrite the typeString? It's coupled with other logic, meanwhile the inputType is only used for the keyboard